core/src/scene/scene.h

```diff
@@ -32,6 +33,10 @@ class Scene {
     Scene();
     ~Scene();
 
+    using ArrayValue = std::vector<double>;
+    using Value = variant<none_type, bool, double, std::string, ArrayValue>;
+    std::unordered_map<std::string, Value> m_globals;
```

Comment:
Ohh! I meant it to be private. Forgot to put it back with accessor! Will update.